### ISSUE TITLE
Open resources from other packages

### ISSUE THREAD

**Issue by EugineKidoz** [facebook/fresco#379]
Please add support for Loading Uri image from resources of other package name (App).

Example :   android.resource://" + some_packageName + "/" + identifier /resource id

**Comment by kunny** [comment]
Use following instead:
```java
Uri uri = new Uri.Builder()
        .scheme(UriUtil.LOCAL_RESOURCE_SCHEME)
        .path(String.valueOf(resId))
        .build();
imageView.setImageURI(uri);
```
Also mentioned at #261, #357.

**Comment by EugineKKK** [comment]
Hello. 
This answer doesn't work!.
What you provided is loading resource from local (Current package name).
That is NOTwhat i meant, i meant loading some resource from external package name, external application, NOT my own.

Some example of loading without using fresco.

final String packName = "com.example2.app";
    String mDrawableName = "app_icon";

    try {
        PackageManager manager = getPackageManager();
        Resources mApk1Resources = manager.getResourcesForApplication(packName);

        int mDrawableResID = mApk1Resources.getIdentifier(mDrawableName, "drawable",packName);

        Drawable myDrawable = mApk1Resources.getDrawable( mDrawableResID );

        if( myDrawable != null )
            TEST.setBackgroundDrawable(myDrawable );

    }
    catch (NameNotFoundException e) {
        // TODO Auto-generated catch block
        e.printStackTrace();
    }

What needed is the ability to load such Resources/Images thru Fresco lib so it will go thru the pipeline and will be properly cached.

It should be a quick add on..
String path = "android.resource://" + other app apckagename + "/" + resourceId;
BufferedInputStream inputStream = null;
               inputStream = new BufferedInputStream(mContext.getContentResolver().openInputStream(uri));
               if (inputStream != null)
               {
                   Bitmap bitmap = BitmapFactory.decodeStream(inputStream, new Rect(-1, -1, -1, -1), null);
                   Store bitmap to fresco cache...
                   inputStream.close();
               }

**Comment by dominic69** [comment]
A much needed feature...

**Comment by vipulasri** [comment]
@tyronen your solution 
<code>
Uri uri = new Uri.Builder()
        .scheme(UriUtil.LOCAL_RESOURCE_SCHEME)
        .path(String.valueOf(resId))
        .build();
imageView.setImageURI(uri);
</code>

not working.

**Comment by hhuimie** [comment]
+1

**Comment by niklee** [comment]
+1 
It's important feature for loading theme resource from plugin-in App.

**Comment by harish2704** [comment]
+1
I was looking for a solution to build a custom "Share Modal Dialog" using React-native.

**Comment by davidliu** [comment]
+1. Not sure why this got closed.

**Comment by laozhao1005** [comment]
+1

**Comment by xdyuchen** [comment]
+1, i want use fresco to cache application icons

**Comment by lambdapioneer** [comment]
Hey everyone, this should be fixed with 4326935ca115422a9aa77505e66cc68bbc76a76c and 4326935ca115422a9aa77505e66cc68bbc76a76c that I've landed last week. This will be available with the next 1.3.0 release.

**Comment by lambdapioneer** [comment]
Released yesterday with https://github.com/facebook/fresco/releases/tag/v1.3.0
